app/src/main/java/np/com/naxa/factsnepal/ProfileInformationActivity.java

```diff
@@ -0,0 +1,165 @@
+import android.widget.EditText;
+import android.widget.Spinner;
+
+import com.jakewharton.rxbinding2.widget.RxAdapterView;
+import com.jakewharton.rxbinding2.widget.RxTextView;
+
+import java.text.ParseException;
+
+import io.reactivex.BackpressureStrategy;
+import io.reactivex.Flowable;
+import io.reactivex.Observable;
+import io.reactivex.functions.Function;
+import io.reactivex.functions.Function6;
+import io.reactivex.subscribers.DisposableSubscriber;
+import np.com.naxa.factsnepal.common.BaseActivity;
+import np.com.naxa.factsnepal.feed.list.FeedListActivity;
+import np.com.naxa.factsnepal.utils.ActivityUtil;
+
+public class ProfileInformationActivity extends BaseActivity implements View.OnClickListener {
+
+    private EditText etWard;
+    private EditText etDistrict;
+    private Button btnNext;
+    private EditText etDob;
+    private Spinner spinnerGender;
+
+
+    private DisposableSubscriber<Boolean> disposableObserver = null;
+    private Flowable<CharSequence> genderChangeObservable;
+    private Flowable<CharSequence> gpsChangeObservable;
+    private EditText etProvience;
+    private EditText etMunicipality;
+    private EditText etStreet;
+    private MaterialButton btnSkip;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_profile_information);
+        initView();
+        setupToolbar();
+        makeUIObservable();
+    }
+
+    private void initView() {
+        etDob = findViewById(R.id.et_dob);
+        spinnerGender = findViewById(R.id.spinner_gender);
+        etWard = findViewById(R.id.et_ward);
+        etDistrict = findViewById(R.id.et_district);
+        etProvience = findViewById(R.id.et_province);
+        etMunicipality = findViewById(R.id.et_municipality);
+        etStreet = findViewById(R.id.et_street);
+        btnSkip = findViewById(R.id.btn_skip);
+        btnSkip.setOnClickListener(this);
+        btnNext = (MaterialButton) findViewById(R.id.btn_next);
+        btnNext.setOnClickListener(this);
+
+    }
+
+    private void makeUIObservable() {
+        Flowable<CharSequence> dobChangeObservable = RxTextView.textChanges(etDob).skip(1).toFlowable(BackpressureStrategy.LATEST);
+        Flowable<CharSequence> wardChangeObservable = RxTextView.textChanges(etWard).skip(1).toFlowable(BackpressureStrategy.LATEST);
+        Flowable<CharSequence> districtChangeObservable = RxTextView.textChanges(etDistrict).skip(1).toFlowable(BackpressureStrategy.LATEST);
+        Flowable<CharSequence> provinceChangeObservable = RxTextView.textChanges(etProvience).skip(1).toFlowable(BackpressureStrategy.LATEST);
+        Flowable<CharSequence> municipalityChangeObservable = RxTextView.textChanges(etMunicipality).skip(1).toFlowable(BackpressureStrategy.LATEST);
+        Flowable<CharSequence> streetChangeObservable = RxTextView.textChanges(etStreet).skip(1).toFlowable(BackpressureStrategy.LATEST);
```

Comment:
comment this line of code and can keep this as a reference for future use.

Reply:
I agree with you